doc/manuals/admin/management_api.md

```diff
@@ -8,7 +8,21 @@ API for management that allows to change the log level and the trace levels
 To change the log level:
 
 ```
-curl --request PUT <host>:<port>/admin/log?level=<FATAL|ERROR|WARNING|WARN|INFO|DEBUG|NONE>
+curl -X PUT <host>:<port>/admin/log?level=<FATAL|ERROR|WARN|INFO|DEBUG|NONE>
```

Comment:
Using -X instead of --request, to align with other parts of the documentation.

NTC (just informative)